### PR TITLE
JK BMS V1.3 Support / SOC Calculations based on Min/Max Cell Voltage

### PR DESCRIPTION
**Pull request by drakesoft**
JK BMS V1.3 Protocol RS485 Support for JK-B1A24S  / JK-B2A24S Balancer
 SOC Calculations based on Min/Max Cell Voltage

### CODE REVIEW

File: include/defines.h [hunk 1]
```diff
@@ -301,6 +303,12 @@ enum serialRxTxEn_e {serialRxTx_RxTxDisable, serialRxTx_TxEn, serialRxTx_RxEn};
 #define ID_PARAM_BTDEV_DEACTIVATE                126
 
 #define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC100_CELL_VOLTAGE 127
+#define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE 128
+
+#define ID_PARAM_TEMP_ALARM_TEMP_QUELLE          129
+#define ID_PARAM_TEMP_ALARM_BMS_QUELLE           130
+
+#define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE 128
```
Comment (shining-man) [inline include/defines.h:312]: The parameters ID_PARAM_TEMP_ALARM_TEMP_QUELLE and ID_PARAM_TEMP_ALARM_BMS_QUELLE originally had the IDs 128 and 129. If existing IDs are changed, the settings are incorrect at the user. Therefore, please do not change any IDs. 
The new parameter ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE should therefore have the ID 130.

In addition, the parameter ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE is duplicated.

Maybe the problem was caused by my last commit.

File: platformio.ini
Comment (shining-man) [inline platformio.ini:1]: Please leave the platformio.ini in its original state and do not replace it with your own board.

**Review by shining-man — CHANGES_REQUESTED** [review]
I have looked at the changes and commented in two places.

**Comment by drakesoft** [comment]
I fixed the issues you mentioned.

**Comment by shining-man** [comment]
Can you possibly add the BMS to the table in BmsData.h? There you can see which data the BMS delivers.

I have seen that you do not fill the bmsErrors and bmsStateFETs or not correctly. This means that the BMS can only be used correctly for monitoring. I will therefore add a comment that the BMS can only be used for monitoring. 
If the BMS is also to be used for charge control, etc., it also needs all the necessary data (bmsErrors, bmsStateFETs, ...).

**Pull request merged**